modules/@apostrophecms/event-page/views/index.html

```diff
@@ -0,0 +1,23 @@
+{% extends data.outerLayout %}
+{% import "filters.html" as filters %}
+{% block title %}{{ data.page.title }}{% endblock %}
+
+{% block main %}
+  <h2>Filters</h2>
+  {% render filters.render({
+    filters: data.piecesFilters,
+    query: data.query,
+    url: data.page._url
+  }) %}
+  <h2>Events</h2>
+  {% for piece in data.pieces %}
+    <article class="apos-event-preview">
```

Comment:
I think we should really remove all these classes. We don't provide starter styles and I would not expect anyone to use these templates as-is. They're primarily starter files to replace in the project.

Reply:
Fine with removing the classes.